corgi/collectors/errata_tool.py

```diff
@@ -75,40 +84,82 @@ def load_et_products(self):
                     defaults={"name": product_version["name"], "product": et_product},
                 )
                 if created:
-                    logger.info("Created ET ProductVersion: %s", et_product_version.name)
+                    logger.info(f"Created ET ProductVersion: {et_product_version.name}")
+
+    def load_product_versions(self) -> None:
         for pv in CollectorErrataProductVersion.objects.get_queryset():
             pv_details = self.get(f"api/v1/products/{pv.product.et_id}/product_versions/{pv.et_id}")
-            brew_tags = [t.removesuffix("-candidate") for t in pv_details["data"]["brew_tags"]]
+            pv_detail_data = pv_details["data"]
+            brew_tags = [t.removesuffix("-candidate") for t in pv_detail_data.pop("brew_tags")]
```

Comment:
Perhaps just creating a constant for "-candidate" will give a maintainer a clue that the 2 functions both adjust the raw values? I think it makes sense to adjust the raw values in both function as we currently do. One is brew_tag data coming from ET, another is brew_tag data coming from prod_defs. 

You did make me realize that we also should probably remove the '-candidate' suffix from the new load_et_releases function as well, so that brew tag matching can behave the same between product models and collector models. I'll adjust that.

Brew tags in ET have '-candidate' tags because Engineers never attached released builds to an Errata. ET switches the brew tags to '-released' once they are attached to an Erratum and released. That's why the ET data and prod_defs data differ. Some products don't use the brew tag candidate and released suffixes at all.